### PR TITLE
TOC

### PR DESCRIPTION
**Pull request by boxdxnx**
Added table of contents for better document navigation

### CODE REVIEW

**Comment by d1mnewz** [comment]
The document itself has formatting issues. This PR makes sense to me because I had to introduce TOC at some moment of time.
However 
```
[_^ back to TOC^_](#table-of-contents)
```
this constantly repeating line seems to add more noise than value. I would be happy to accept this PR without it. 
Also, as a side issue, we can fix formatting in the document since most likely it doesn't correspond to the syntax of [MD-language](https://daringfireball.net/projects/markdown/syntax). I can do that by myself using VS Code markdown extensions which highlights syntax errors, otherwise, you can do that yourself but in a separate PR, please.

Thanks for your contribution!

**Comment by boxdxnx** [comment]
@d1mnewz `[_^ back to TOC^_](#table-of-contents)` deleted !

File: README.md [hunk 1]
```diff
@@ -236,6 +263,8 @@ P.P.S. The philosophy of hiring is mostly "hiring the best ones" - I agree to th
 # What you can ask the company
 Don't ask them if you just wanna make an impression because you will. And it's gonna be a freaking bad impression - asking by template without any enthusiasm. What I really like to do is to start doing 96 and after recruiter asked you "Where do you see yourself in 5 years" it's totally reasonable to ask where the company sees itself in 5 years.
 
+[_^ back to TOC^_](#table-of-contents)
```
Comment (d1mnewz) [inline README.md]: @babayK0 Why should we leave this one?

**Review by d1mnewz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (boxdxnx) [reply README.md]: @d1mnewz my bad. wait sec

**Review by d1mnewz — APPROVED** [review]

**Comment by d1mnewz** [comment]
Nice! Thanks for your contribution!

**Pull request merged**